### PR TITLE
Browsing

### PR DESCRIPTION
**Pull request by bahrieinn**
All good?

### CODE REVIEW

File: app/controllers/questions_controller.rb [hunk 1]
```diff
@@ -13,17 +13,20 @@ def new
...
-    # Create callback in Question model to parse tags
-    p params[:question][:tags]
-
-    # @tags = params[:tags].map { |tag| tag.strip }
-    # @tags.each do |tag|
-    #   @question.tags.find_or_create_by(name: tag)
-    # end
-    # redirect_to @question
+    if current_user
+      @user = current_user
```
Comment (kelmerp) [inline app/controllers/questions_controller.rb:17]: if you have current_user, you can just do current_user.questions.new....

File: app/controllers/answers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class AnswersController < ApplicationController
+
+  def create
+    if true
```
Comment (tarynsauer) [inline app/controllers/answers_controller.rb:4]: Change condition to check for current_user.

File: app/controllers/answers_controller.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      @user = User.first
```
Comment (tarynsauer) [inline app/controllers/answers_controller.rb:5]: this is also for testing?  does it need to be in pull req?  :poop:

File: app/controllers/answers_controller.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      @question = Question.find(params[:question_id])
+      @answer = @question.answers.create(content: params[:answer][:content])
+      @answer.update(user_id: @user.id)
+      # @answer = Answer.new(content: params[:answer][:content])
+      # p @answer
+      redirect_to @question
+    else
+      redirect_to @question
```
Comment (tarynsauer) [inline app/controllers/answers_controller.rb:13]: should there be some alert here -- eg. 'must be logged in to create answer?

File: app/views/users/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+<% unless @user.questions.empty? %>
+  <ul>
+  <% @user.questions.each do |question| %>
+    <li>
+      <%= question.content %>
+    </li>
+  <% end %>
+  </ul>
+<% end %>
```
Comment (tarynsauer) [inline app/views/users/show.html.erb:14]: are we also going to add followed questions to user profile?

File: app/views/welcome/index.html.erb [hunk 6]
```diff
@@ -1,6 +1,18 @@
-<h1>Hello world</h1>
+<h1>Ask Deebz</h1>
```
Comment (tarynsauer) [inline app/views/welcome/index.html.erb:1]: a z!?  frivolous commentz count: 1

**Pull request merged**